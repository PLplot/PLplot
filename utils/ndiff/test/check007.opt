--absolute-error 1.e-14
