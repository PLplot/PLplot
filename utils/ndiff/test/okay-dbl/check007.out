### Maximum absolute error in matching lines = 8.88e-15 at line 31 field 4
